vendors/st/boards/stm32l475_discovery/aws_demos/config_files/aws_demo_config.h

```diff
@@ -69,4 +69,16 @@
 #define democonfigGREENGRASS_DISCOVERY_TASK_STACK_SIZE    ( configMINIMAL_STACK_SIZE * 22 )
 #define democonfigGREENGRASS_DISCOVERY_TASK_PRIORITY      ( tskIDLE_PRIORITY )
 
+#define democonfigMEMORY_ANALYSIS
+
+#ifdef democonfigMEMORY_ANALYSIS
+    #define demoMEMORY_ANALYSIS_STACK_DEPTH_TYPE    UBaseType_t
+    #define demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE( x )     xPortGetMinimumEverFreeHeapSize()
```

Comment:
Same here. What's the intention for input (x)?

Reply:
My intention was for when the kernel heap gets updated to https://sourceforge.net/p/freertos/code/HEAD/tree/trunk/FreeRTOS/Source/portable/MemMang/heap_4.c#l442. Should this just be done later?

Reply:
I see. Prefer to update macro definition to match the function getting called.

(Code refactoring is always expected. By leaving the code as-is for now we are implicitly saying the later half needs to be updated, which no one can guarantee when that's going to happen... Less desired... )